pdg/src/info.rs

```diff
@@ -0,0 +1,12 @@
+use crate::Graphs;
+
+#[derive(Hash, Clone, PartialEq, Debug)]
+pub struct NodeInfo {}
```

Comment:
Add some documentation here.

Part of it can be a general overview of what it's for, and part of it can be what's it's intended to contain.